Gemfile.lock

```diff
@@ -176,50 +179,50 @@ GEM
     net-ssh (3.2.0)
     netrc (0.11.0)
     newrelic_rpm (3.16.0.318)
+    nio4r (1.2.1)
     nokogiri (1.6.8)
       mini_portile2 (~> 2.1.0)
       pkg-config (~> 1.1.7)
     parser (2.3.1.2)
       ast (~> 2.2)
-    paul_revere (2.0.0)
-      rails (~> 4.0)
+    paul_revere (2.1.0.rc1)
```

Comment:
paul_revere seems to be [abandoned](https://github.com/thoughtbot/paul_revere/issues/35) from what I can tell, can we replace it with something else?